data/bookkeeping/block_test.go

```diff
@@ -1216,3 +1226,222 @@ func TestAlive(t *testing.T) {
 		t.Errorf("expired transaction alive %v", header)
 	}
 }
+
+func TestNextBaseFee(t *testing.T) {
```

Comment:
BaseFee can move by 50% up or down each round. We should decide whether we want it to be able to move that fast.

In comparison, Ethereum can only change 12.5% per block (and their blocks are also slower).